cmd/immuadmin/command/backup_test.go

```diff
@@ -828,7 +828,7 @@ func TestCommandlineBck_Register(t *testing.T) {
 func TestNewCommandLineBck(t *testing.T) {
 	cml, err := newCommandlineBck(immuos.NewStandardOS())
 	assert.IsType(t, &commandlineBck{}, cml)
-	assert.Nil(t, err)
+	assert.NoError(t, err)
```

Comment:
shall we use require instead? e.g. `require.NoError(t, err)`

Reply:
That would depend on the use-case, assert will not stop the current test where require will - in this cleanup I did not analyze all the cases thus I left asserts / requires as they were before